File: Young Physicist.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int sum_x = 0, sum_y = 0, sum_z = 0, t; cin >> t;
    while (t--) {
        int x, y, z; cin >> x >> y >> z;
        sum_x += x;
        sum_y += y;
        sum_z += z;
    }
    if (sum_x == 0 && sum_y == 0 && sum_z == 0)
        cout << "YES";
    else
        cout << "NO";
    cout << endl;

    return 0;
}

File: Java/YoungPhysicist.java

import java.util.*;

public class YoungPhysicist {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
	int n ;
	n= in.nextInt();
	int sumx=0, sumy=0 , sumz=0;
	for(int i=0 ; i < n ;i++) {
		sumx += in.nextInt();
		sumy += in.nextInt();
		sumz += in.nextInt();
		
	}
	
	if(sumx==0&&sumy==0&&sumz==0) {
		System.out.println("YES");
	}else {
		System.out.println("NO");
	}
	in.close();
	}

}

